### PR TITLE
Tutorial tweaks

### PR DESCRIPTION
**Pull request by alexweininger**

### CODE REVIEW

File: resources/TrialApp.didact.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Welcome to your Trial App Didact Tutorial
+# Welcome to your Trial App
```
Comment (nturinski) [inline resources/TrialApp.didact.md:1]: Not Trial App Experience?

**Comment by alexweininger** [comment]
image
image
image

File: resources/TrialApp.didact.md (hunk 1 above)
Comment (alexweininger) [reply resources/TrialApp.didact.md:1]: Matt requested this, I don't know if either of us considered putting "Trial App Experience"

File: resources/TrialApp.didact.md [hunk 2]
```diff
@@ -42,7 +44,7 @@ For example, change line 26 from: `<h1> Welcome to Azure! </h1>` to this: `<h1>
 
 ![Deploy complete notification](./trialAppTutorial/deployCompleteScreenshot.png)
 
-You can also browse your website by right-clicking your trial app and selecting `Browse`.
+You can also browse your website by right-clicking your trial app and selecting `Browse website`.
```
Comment (nturinski) [inline resources/TrialApp.didact.md:47]: casing on Website in the Browse Website

File: resources/TrialApp.didact.md [hunk 3]
```diff
@@ -11,13 +11,15 @@ This tutorial will show you how to make and deploy changes to your Azure App Ser
 ![Trial app view](./trialAppTutorial/viewTrialAppScreenshot.png)
 
 2. Right-click on the trial app.
-3. Select `Clone`.
+3. Select `Clone Trial App...`.
```
Comment (nturinski) [inline resources/TrialApp.didact.md:14]: I think you should get rid of the period at the end here.  It looks weird preceded with `...`

File: resources/TrialApp.didact.md (hunk 2 above)
Comment (nturinski) [reply resources/TrialApp.didact.md:47]: I think the period looks here too fwiw

File: resources/TrialApp.didact.md (hunk 1 above)
Comment (nturinski) [reply resources/TrialApp.didact.md:1]: Hahaha, I'm just teasing a little since it was in your presentation

**Pull request merged**